libraries/chain/fork_database.cpp

```diff
@@ -719,8 +722,7 @@ namespace eosio::chain {
       auto in_use_value = in_use.load();
       // check that fork_dbs are in a consistent state
       if (!legacy_valid && !savanna_valid) {
-         wlog( "fork_database is in a bad state when closing; not writing out '${filename}', legacy_valid=${l}, savanna_valid=${s}",
-               ("filename", fork_db_file)("l", legacy_valid)("s", savanna_valid) );
+         ilog("No fork_database to persist, not writing out: ${f}", ("f", fork_db_file));
```

Comment:
Since we are not writing out, we don't need to log the file name `fork_db_file`.

Reply:
I thought it didn't hurt to include the file. If you think it is confusing, I can remove it.

Reply:
I think it is a little confusing. The user might think an empty file is there. But  this is very minor, you don't need to change.

Reply:
Makes sense, I'll remove it